src/test/java/teammates/test/cases/ui/browsertests/StudentCourseJoinConfirmationPageUiTest.java

```diff
@@ -70,12 +71,17 @@ private void testJoinNewConfirmation() {
         studentHomePage = createCorrectLoginPageType(browser.driver.getPageSource())
                            .loginAsStudent(TestProperties.inst().TEST_STUDENT1_ACCOUNT,
                                                   TestProperties.inst().TEST_STUDENT1_PASSWORD);
-        studentHomePage.verifyStatus(
-                String.format(Const.StatusMessages.STUDENT_COURSE_JOIN_SUCCESSFUL, courseId));
+        
+        String expectedStatus = String.format(Const.StatusMessages.STUDENT_COURSE_JOIN_SUCCESSFUL, "[" + courseId + "] " + courseName) + '\n' 
+                                + String.format(Const.StatusMessages.HINT_FOR_NO_SESSIONS_STUDENT, "[" + courseId + "] " + courseName) + '\n'  
+                                + "Meanwhile, you can update your profile here.";
```

Comment:
Hey @kanghj, just a clarification, we expect to see Const.StatusMessages.STUDENT_UPDATE_PROFILE here right? That constant's value seems a little different from the value you are checking here. Does the test pass?